packages/contracts/contracts/mocks/MockFollowValidationModuleBase.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {IFollowModule} from '../interfaces/IFollowModule.sol';
+import {ILensHub} from '../interfaces/ILensHub.sol';
+import {Errors} from '../libraries/Errors.sol';
+import {Events} from '../libraries/Events.sol';
+import {ModuleBase} from '../ModuleBase.sol';
+import {IERC721} from '@openzeppelin/contracts/token/ERC721/IERC721.sol';
+
+/**
+ * @title FollowValidationModuleBase
+ * @author Lens Protocol
+ *
+ * @notice This abstract contract adds a simple non-specific follow validation function.
+ *
+ * NOTE: Both the `HUB` variable and `_checkFollowValidity()` function are exposed to inheriting
+ * contracts.
+ *
+ * NOTE: This is only compatible with COLLECT & REFERENCE MODULES.
+ */
+abstract contract FollowValidationModuleBase is ModuleBase {
```

Comment:
Why do we need something in the domain of Follow modules?

Reply:
The module check some data on this contract.  So I stubbed it to return true.

Reply:
deleted this in favor of mocking in the tests.